updatecli/updatecli.d/terraform-aws-modules-ecr-aws.yaml

```diff
@@ -0,0 +1,38 @@
+scms:
+  default:
+    kind: github
+    spec:
+      user: "{{ .github.user }}"
+      email: "{{ .github.email }}"
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.repository }}"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      branch: "{{ .github.branch }}"
+
+sources:
+  latestECRVersion:
+    name: "Get latest version of terraform-aws-modules/secrets-manager/aws"
+    kind: "terraform/registry"
+    spec:
+      module: "terraform-aws-modules/secrets-manager/aws"
+      constraint: ">1.3.1"
+
+targets:
+  updateECRModule:
+    name: "Update Secrets-Manager module version in ecr.tf"
+    kind: "file"
+    sourceid: latestECRVersion
+    spec:
+      file: "ecr.tf"
+      matchPattern: "version = \"[0-9.]+\""
+      replacePattern: "version = \"{{ .Source }}\""
```

Comment:
Please use the native `hcl` kind (like in https://github.com/jenkins-infra/terraform-aws-sponsorship/blob/b25e94c3f80b3e856fc9de386222881c6d42d6a2/updatecli/updatecli.d/irsa-role.yaml#L25-L33)